metro.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  resetCache: true,
+};
```

Comment:
❓ What's the reason for adding this?

Reply:
Without this, the metro server does not pick up changes made to the env file, because the env file is not part of the files that metro looks for changes on.